lib/showoff.rb

```diff
@@ -858,7 +858,6 @@ def update_commandline_code(slide)
           lines = out.split("\n")
           if lines.first.strip[0, 3] == '@@@'
             lang = lines.shift.gsub('@@@', '').strip
-            pre.set_attribute('class', 'highlight')
```

Comment:
I'm confused as to what CommonMarker does differently that makes this not work. The rest of the code can only work if it's got nested `<pre><code>` blocks anyway. I'll poke a little and see what's being generated.